projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/saved-cart/b2b-saved-cart.e2e-spec.ts

```diff
@@ -100,6 +100,33 @@ context('B2B - Saved Cart', () => {
         it('should be able to save the active cart and view it in the listing page', () => {
           savedCart.saveActiveCart();
         });
+
+        it('should create cart, save it and restore with a clone with default form values twice and then compare row in saved cart listing page', () => {
+          savedCart.saveActiveCart();
+
+          cy.get('cx-saved-cart-list .cx-saved-cart-list-table tr')
+            .first()
+            .find('.cx-saved-cart-list-cart-id > .cx-saved-cart-list-value')
+            .then((cartId) => {
+              const cartCode = cartId.text();
+
+              savedCart.manuallyRestoryCartFromListingPage(cartCode);
+              savedCart.visitCartPage();
+
+              cy.window()
+                .then((win) =>
+                  JSON.parse(
+                    win.localStorage.getItem('spartacus⚿powertools-spa⚿cart')
+                  )
+                )
+                .then(({ activeCartCode }) => {
+                  savedCart.saveActiveCart(true, false);
```

Comment:
what's the point of passing true for manual and false for checking the alert. When we save it, it shows so why not check it?

Also, what's the point of these params to check? We should always assert when we save a cart that it shows the global message and was saved

--------
Since the saveActiveCart function is asserting from another place, maybe you can update the function to be more 'dynamic' when doing the assertions. I'll leave it up to Kate to decide, but either way would be fine as you have already created other functions to 'bypass' this.

Reply:
Basically, for this case, there should have 2 saved carts in the listing page now. It would be great to check if it was saved and that there are 2 saved carts now